src/mscorlib/corefx/Interop/Unix/System.Native/Interop.GetRandomBytes.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I think you can put this under src\mscorlib\src\Interop. We should get rid of everything under corefx.